batch_training/model_from_scratch/app_token.json

```diff
@@ -0,0 +1 @@
+{"function_id":"JXEw8oBU2VLWAcePHtNJcU","function_token":"eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjM1MzM4NDQ3MzIsImlhdCI6MTY2OTEzMjY4NCwiaXNzIjoiZ2l0aHVifDI2MzY4MjQ5Iiwic2NvcGUiOiJmdW5jdGlvbjppbnZva2UiLCJzdWIiOiJKWEV3OG9CVTJWTFdBY2VQSHROSmNVIn0.r_ABN4CpEuQRsHdXiCCJuJ9ENQraDE69fxt8afCBsmubRaPgLv4yrJPhV10XZ-dVTijmllqnG9bMpONsmcMOoKQfZ8YrWe9ztd5s4iuwiKKIuG7AOvC7QMhQ_sCchZrLy3DGedUXa4COAs_aq2FkQWzoLx6hk6kMQaqmkhkYilnZOh9x5Oa0nUKfohFPtAWrM84RFB-0UzPFon4ptViOVA3qnqWMVxKFxuw6VJo8g7X-epoghEr3R2_oUkmyCTnX6invzB6DwdNOeAE9Vum_T0l0tK-F0GvujXI35VXH74XOInYzCGcjdepmxFkuyHj2vLl0bmExWb7yDvD67xVlVA","function_checksum":"e6834cdf179c610366315f24808a4377b9811451656d4a25bf82ff013a01ffab"}
```

Comment:
Is this function public?

Reply:
good catch, I made the function public and set the token expiry to be a long time from now